localstack/services/awslambda/lambda_executors.py

```diff
@@ -387,6 +393,10 @@ def prime_docker_container(self, runtime, func_arn, env_vars, lambda_cwd):
 
             return ContainerInfo(container_name, entry_point)
 
+    def get_host_path_for_path_in_docker(self, path):
```

Comment:
Can we extract this code into a shared util function (on global scope), to avoid duplication with line 614?

Reply:
yep no problem, I thought that would be the case ☺️ just didn’t know where to put it. I saw a utils folder, in one of those files or a new one?

Reply:
I think we could also just put it towards the end of the same file. There's actually already a `Util` class where we could add it. Thanks

Reply:
Ah awesome, didn't see that there. Pushed changes now 👍